### PR TITLE
Initial version of the JetBrains Inspect Code Issue Provider Addin

### PR DESCRIPTION
**Pull request by pascalberger**
Initial version of the JetBrains Inspect Code Issue Provider Addin

### CODE REVIEW

**Comment by gep13** [comment]
Looks like Cake.Figlet doesn't like it when the text is too long:

image

😄

**Comment by pascalberger** [comment]
@gep13 Yeah, saw that. Think I'll need to use a shorter title than the addin name

**Comment by gep13** [comment]
>@pascalberger said...
Think I'll need to use a shorter title than the addin name

I would need to check if that is "allowed" currently.  The title is used in a couple places.  Perhaps we need a separate property (which defaults to the same title) which can be overridden.

**Comment by pascalberger** [comment]
@gep13 Currently `BuildParameters.Title` seems to only be used for Figlet and the following `Information` output. See [here](Title .cake)

**Comment by gep13** [comment]
@pascalberger correct.  However, within the SetParameters method, the value of title is used to infer a number of other properties:

image

**Comment by pascalberger** [comment]
@gep13 Ah, missed these :)

File: src/Cake.Prca.Issues.InspectCode.Tests/packages.config [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <package id="Cake.Testing" version="0.18.0" targetFramework="net452" />
+  <package id="Shouldly" version="2.8.2" targetFramework="net452" />
+  <package id="xunit" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.abstractions" version="2.0.1" targetFramework="net452" />
+  <package id="xunit.assert" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.core" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.extensibility.core" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.extensibility.execution" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.runner.visualstudio" version="2.2.0" targetFramework="net452" developmentDependency="true" />
+</packages>
```
Comment (stffabi) [inline src/Cake.Prca.Issues.InspectCode.Tests/packages.config:14]: StyleCop.Analyzers

File: src/Cake.Prca.Issues.InspectCode/packages.config [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Cake.Core" version="0.18.0" targetFramework="net452" />
+  <package id="Cake.Prca" version="0.1.0-unstable0011" targetFramework="net452" />
+</packages>
```
Comment (stffabi) [inline src/Cake.Prca.Issues.InspectCode/packages.config:5]: Add StyleCop.Analyzers

**Review by stffabi — APPROVED** [review]

File: src/Cake.Prca.Issues.InspectCode.Tests/packages.config (hunk 1 above)
Comment (pascalberger) [reply src/Cake.Prca.Issues.InspectCode.Tests/packages.config:14]: Already added https://github.com/cake-contrib/Cake.Prca.Issues.InspectCode/issues/2 for this

**Pull request merged**